subjects/cybersecurity/hole-in-bin/README.md

```diff
@@ -1,26 +1,44 @@
-# Hole in Bin
+# Hole-In-Bin
+
+<center>
+<img src="./pictures/meme.png?raw=true" style = "width: 600px !important; height: 737px !important;"/>
+</center>
 
 ### Objective
 
-The objective of the project is that you are comfortable in reverse engineering on a windows environment, because many systems (Bank ATMs, SCADA..) can run on windows.
+This exercise is designed to test your skills and understanding of binary exploitation and reverse engineering. You will need to work through a series of binary exploitation challenges using a provided virtual machine.
 
-### Guidelines
+### Setup
 
-You will have the exercise of creating a patch that will bypass the security of the [program](./resources/hole-in-bin.exe) and display "please withdraw your money" in the color green, rather than an error message..
+1. Download the virtual machine image [hole-in-bin.ova](https://assets.01-edu.org/cybersecurity/hole-in-bin/hole-in-bin.ova). This image contains all the binaries you will need to exploit.
 
-### Advice
+SHA1: 7db09b7a8fdfe25c286561dfa7ca5b50718bd60c
```

Comment:
This could be highlighted and labeled so that the students can understand what is it for

Reply:
It's not required to understand it, It's for people who know what SHA1 is and wanna check it

Reply:
In my opinion it should be labeled so that users with less experience know what is it for.